client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/index.tsx

```diff
@@ -44,32 +46,38 @@ export default function ItemPreviewPaneHeader( {
 	return (
 		<div className={ classNames( 'item-preview__header', className ) }>
 			<div className="item-preview__header-content">
-				<SiteFavicon
-					blogId={ itemData.blogId }
-					fallback={ siteIconFallback }
-					color={ itemData.color }
-					className="item-preview__header-favicon"
-					size={ size }
-				/>
+				{ isSiteData && (
+					<SiteFavicon
+						blogId={ itemData.blogId }
+						fallback={ siteIconFallback }
+						color={ itemData.color }
+						className="item-preview__header-favicon"
+						size={ size }
+					/>
+				) }
 				<div className="item-preview__header-info">
 					<div className="item-preview__header-title-summary">
 						<div className="item-preview__header-title">{ itemData.title }</div>
 						<div className="item-preview__header-summary">
-							<Button
-								variant="link"
-								className="item-preview__header-summary-link"
-								href={ itemData.url }
-								target="_blank"
-							>
-								<span>
-									{ itemData.subtitle }
-									<Icon
-										className="sidebar-v2__external-icon"
-										icon={ external }
-										size={ extraProps?.externalIconSize || ICON_SIZE_SMALL }
-									/>
-								</span>
-							</Button>
+							{ isSiteData ? (
+								<Button
+									variant="link"
+									className="item-preview__header-summary-link"
+									href={ itemData.url }
+									target="_blank"
+								>
+									<span>
+										{ itemData.subtitle }
+										<Icon
+											className="sidebar-v2__external-icon"
+											icon={ external }
+											size={ extraProps?.externalIconSize || ICON_SIZE_SMALL }
+										/>
+									</span>
+								</Button>
+							) : (
+								itemData.subtitle
+							) }
```

Comment:
And here is something similar to the above suggestion, `subtitleWithLink` or even better, just check if `itemData.url`  is not empty/undefined to determine to use the Button+Icon or just render the subtitle.